rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -1091,6 +1095,12 @@ def main():
         type=str,
         help='Specify a desired branch of the corresponding git',
         default='master')
+    parser.add_argument(
+        '--ticket_id',
+        type=str,
+        help='Approved Ticket ID for cloud-spend',
+        required=False
```

Comment:
If we are going to add this, required has to be True

Reply:
>If we are going to add this, the required has to be True

We need this `ticket-id` if our initial funding is exhausted.
As such, I think this should be `False`

Reply:
Looping in @smalleni @dry923 ^^^ can you weigh in as well?

Reply:
Hi 👋 @dry923 @smalleni ptal..